### PR TITLE
fix(#750): check babala reason is not NaN

### LINKED ISSUE #750: bug: notification type is "nan"
## Bug Definition
In other notifications, a phrase like "enkaz" comes in, but "nan" comes up in the case I come across.

** discord username: @kemdetroit#4551 **

## Bug environment
afetharita.com

## Describe how you are producing the bug step by step

1. Go to ['this link'](https://afetharita.com/?lat=36.775729742227625&lng=36.52130126953126&zoom=10&id=706163)
2. Look left side
3. Focus on with blue clored label
4. Detect "nan" 

## Expected Behaviour
If nan is coming, it should not appear at all, or why "nan" is coming should be asked in the backend or a different solution.

## Screen shots
image

## Desktop Information
it doesn't matter

## Mobile Phone Information
it doesn't matter

## Additional Context
no more

### PR DESCRIPTION
**Pull request by eytanfb**
# Description
**closes #750**

If `reason` from the API is `NaN` (which I think will be fixed), do not display the `reason` value
**discord username: eytan#6012**

## Before
image

## After
Screenshot 2023-02-10 at 7 37 20 PM

## Things to check before creating a PR
- [x] I have inspected my topic, checked.
- [ ] If it is a core feature, executed detailed tests.

## Creating PR rules
- [x] PR must be created for an issue with approved tag. Otherwise PR will be rejected.
- [x] Relevant issue number: The issue number related to PR must be attached to head of PR header, after prefix after prefix # must be attached in parenthesis. A header like this could be used "prefix(#issue_number): PR header" .
- [x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules. For example, a title like "docs(#issueId): Add README.md" can be used.
- [x] Related file selection: Only relevant files should be touched and no other files should be affected.
- [x] Format and Lint Suitability : The code should be made in accordance with a certain format standard and examined according to the lint rules.
- [x] Clean commit history: The commits where the changes are made should be clear and organized.
- [x] Opening PR when job is completed: PR should be opened when job is completed and sent for review by other team members.

## Changes

- [ ] A new feature (a change that adds a new feature, not a breaking change)
- [ ] A new refactor (a change that is not a breaking change, that improves the readability or performance of the code)
- [ ] A breaking change
- [ ] Documentation change

# How were these changes tested?

Tested on Chrome for desktop and Chrome emulator for mobile.

**Test Configuration**:

* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Review by ercumentlacin — DISMISSED** [review]

File: components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx [hunk 1]
```diff
@@ -20,13 +20,17 @@ const FeedChannelBabala = ({
       }
     });
 
+  const isReasonDisplayable = !!reason && reason !== "NaN";
```
Comment (ssibrahimbas) [inline components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx:23]: this does not seem reactive. Can you hold this variable with useState and make it reactive with useEffect?

Comment (ercumentlacin) [reply components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx:23]: Reason yukarıdan props olarak geldiği için bunu state'de tutmaya gerek olmadığını düşünüyorum. Hali hazırda string bir ifade olduğundan farklı bir değer sonrası component re-render olacaktır. Tabi yanlış da değerlendirmiş olabilirim kaynak:

https://beta.reactjs.org/learn/you-might-not-need-an-effect

Comment (cdagli) [reply components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx:23]: Ufak bir oneri, en yukarida `isNaNstring` diye bir method tanimlayip hem 14. satirda hem burada kullanabilir miyiz? Belli ki `nan` da gelebiliyor, `NaN` da gelebiliyor.

**Comment by CyhnYldz** [comment]
It works as expected. Congrats.
@giveerr @eytanfb

**Comment by oguzergul** [comment]
@ercumentlacin I will approve this task after the change requests are completed.

**Review by OrhanOzkercin — DISMISSED** [review]

**Comment by OrhanOzkercin** [comment]
@oguzergul where is the request change. I could not see it

File: components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx (hunk 1 above)
Comment (OrhanOzkercin) [reply components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx:23]: @eytanfb Could you please update this.

File: components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx [hunk 2]
```diff
@@ -20,13 +24,16 @@ const FeedChannelBabala = ({
...
+
   return (
     <>
       <div style={styles.container}>
         <div style={styles.logo_container}>
           <Typography style={styles.logo}>Babala</Typography>
           <div style={styles.chip_container}>
-            {!!reason && <Chip label={reason.toLowerCase()} color="info" />}
+            {isNotNan(reason) && (
+              <Chip label={reason?.toLowerCase()} color="info" />
```
Comment (eytanfb) [inline components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx:35]: eslint didn't let me without the `reason?`

**Comment by eytanfb** [comment]
@oguzergul updated

**Review by cdagli — APPROVED** [review]

**Review by ercumentlacin — APPROVED** [review]

**Review by fdemir — APPROVED** [review]

**Pull request merged**